spread.yaml

```diff
@@ -103,7 +103,7 @@ backends:
                   image: ubuntu-16.04-64
                   workers: 6
             - ubuntu-core-18-64:
-                  image: ubuntu-16.04-64
```

Comment:
This looks a bit unrelated? Fwiw, I'm fine with the change but it does not really matter what the starting point is, the initial image is just used to create a core image that is then blasted over the disk